package.json

```diff
@@ -25,17 +25,7 @@
   },
   "homepage": "https://github.com/visionappscz/eslint-config-visionapps#readme",
   "dependencies": {
-    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-airbnb": "^18.2.0",
     "eslint-plugin-promise": "^4.2.1"
-  },
-  "peerDependencies": {
```

Comment:
`peerDependencies` and `engines` are defined in `eslint-config-airbnb`. If we keep them here we need to keep them in sync with the definition there.

The cost we pay for not having them here is bunch of warnings when running `npm install` in this project, but I think that is worth it.

Reply:
OK